code/processes/rdb.q

```diff
@@ -200,5 +200,6 @@ reload:.rdb.reload
 .rdb.setpartition[]
 
 /-change timeout to zero before eod flush
-.timer.repeat[.eodtime.nextroll-00:01;0W;1D;
+/-GMT offset rounded to nearest 15 mins and added to roll time
+.timer.repeat[.eodtime.nextroll+{00:01*15*"j"$(`minute$x)%15}(.proc.cp[]-.z.p)-00:01;0W;1D;
```

Comment:
I don't think the -00:01 is correct here - it is being subtracted before the casting isn't it? Should be
.eodtime.nextroll+-00:01+{00:01*15*"j"$(`minute$x)%15}(.proc.cp[]-.z.p)
I think